app/models/shift.rb

```diff
@@ -7,6 +7,8 @@ class Shift < ApplicationRecord
   before_destroy :notify_user_of_cancelation, if: -> { user.present? }
   validates :start_at, :duration, presence: true
 
+  scope :claimed, -> { where.not(user_id: nil).reorder(nil) }
```

Comment:
Assume there was a reason for bypassing the default scope? Not saying it's wrong - just makes me wonder why it was bypassed and/or if a default scope should be there in the first place.